spi/core-spi/src/main/java/org/eclipse/dataspaceconnector/spi/event/AssetCreated.java

```diff
@@ -0,0 +1,52 @@
+/*
+ *  Copyright (c) 2022 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Bayerische Motoren Werke Aktiengesellschaft (BMW AG) - initial API and implementation
+ *
+ */
+
+package org.eclipse.dataspaceconnector.spi.event;
+
+import java.util.Objects;
+
+public class AssetCreated extends Event {
+
+    private String id;
```

Comment:
@paullatzelsperger You're right, I was thinking about the `Event` structure, maybe we could keep an `EventEnvelope` that contains an Event field so that we won't mind about field name conflicts. wdyt about this?